ed25519/src/seed.cpp

```diff
@@ -15,7 +22,12 @@
 #endif
 
 void ed25519_create_seed(unsigned char *seed) {
-#ifdef _WIN32
+#ifdef TORRENT_WINRT
+    IBuffer^ seedBuffer = CryptographicBuffer::GenerateRandom(32);
```

Comment:
definately. the modern apis and cx go hand in hand. all the sample code for them use cx